semcore/button/src/index.d.ts

```diff
@@ -29,6 +29,10 @@ export interface IButtonProps extends IBoxProps, INeighborItemProps, IKeyboardFo
   addonLeft?: React.ElementType;
   /** Tag for the right Addon */
   addonRight?: React.ElementType;
+  /** aria-label attribute for a11y
```

Comment:
Why have you put this line outside of `@description`?

Reply:
to be honest idk. I just wanted to show users that this prop will be required in the next release and put in a separate field